KubeArmor/nfpm.yaml

```diff
@@ -0,0 +1,32 @@
+name: "kubearmor"
+arch: "${ARCH}"
+platform: "linux"
+version: "${VERSION}"
+section: "default"
+priority: "extra"
+replaces:
+  - kubearmor
+provides:
+  - kubearmor
+maintainer: "Barun Acharya <barun.acharya@accuknox.com>"
+description: |
+  Cloud-native Runtime Security Enforcement System
+vendor: "kubearmor"
+homepage: "https://kubearmor.com"
+license: "Apache 2"
+contents:
+  - src: ./kubearmor
+    dst: /usr/local/bin/kubearmor
+  - src: ./packaging/kubearmor.conf
+    dst: /etc/sysconfig/kubearmor.conf
+    type: config
+  - src: ./packaging/kubearmor.service
+    dst: /usr/lib/systemd/system/kubearmor.service
+    type: config
+  - src: /usr/local/bin/kubearmor
+    dst: /usr/sbin/kubearmor
```

Comment:
do we need to add kubearmor into /usr/sbin too?

Reply:
> /usr/sbin -> This directory contains program binaries for  system  administration 

So I thought we should put it there too 🤔 I don't think it's necessary